chromium_src/components/favicon/core/favicon_database_browsertest.cc

```diff
@@ -14,6 +14,7 @@
 #include "components/network_session_configurator/common/network_switches.h"
```

Comment:
can you please move this test from `chromium_src` to `browser`? It doesn't belong here.